src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlClientDiagnosticListener.cs

```diff
@@ -91,12 +92,14 @@ public override void OnCustom(string name, Activity activity, object payload)
                             activity.DisplayName = (string)database;
 
                             _ = this.dataSourceFetcher.TryFetch(connection, out var dataSource);
+                            _ = this.connectionStringFetcher.TryFetch(connection, out var connectionString);
```

Comment:
is connectionstring used only in `AddDBUserToActivity`? If yes, then this TryFetch can also be protected under the if `EnableConnectionLevelAttributes` flag.

Reply:
Updated